source/common/runtime/runtime_features.cc

```diff
@@ -91,6 +91,7 @@ RUNTIME_GUARD(envoy_reloadable_features_upstream_allow_connect_with_2xx);
 RUNTIME_GUARD(envoy_reloadable_features_upstream_wait_for_response_headers_before_disabling_read);
 RUNTIME_GUARD(envoy_reloadable_features_use_cluster_cache_for_alt_protocols_filter);
 RUNTIME_GUARD(envoy_reloadable_features_use_http3_header_normalisation);
+RUNTIME_GUARD(envoy_reloadable_features_use_retry_admission_control);
```

Comment:
Added a runtime feature guard to toggle back to the existing `ResourceManager` retry circuit breaking interface. Note that the admission controller will still be created in the cluster info, but will not be used. The feature guard is latched in the router constructor.

This feature guard also serves to decide whether the resource manager or the retry admission control extension gets to write the retry-related circuit breaker stats, which would otherwise be under contention.

The resource manager's retry resource is updated even if this is true, but it isn't consulted prior to allowing a retry.

Reply:
Ideally, this patch needn't any runtime guard. The new behaviors/extensions should only be used when the new limiter is set.